src/Components/Publishing/Sections/Text.tsx

```diff
@@ -57,6 +60,27 @@ export class Text extends Component<Props, State> {
     const { children, color, isContentStart, layout, postscript } = this.props
     const { html } = this.state
 
+    const transform = (node, index) => {
+      if (node.name === "p") {
```

Comment:
Can't have `<div>`s inside of `<p>`s so needed to convert them for the tooltips.